src/LM/langmodel.jl

```diff
@@ -0,0 +1,244 @@
+abstract type Langmodel end
+abstract type gammamodel <: Langmodel end #BaseNgram with Add-one smoothing algo
+abstract type InterpolatedLanguageModel <: Langmodel end #Interpolated language model with smoothing
+
+#DataType MLE
+#Type for providing MLE ngram model scores.
+#Implementation of Base Ngram Model.
+
+struct MLE <: Langmodel
+    vocab::Vocabulary
+end
+
+"""
+    MLE(word::Vector{T}, unk_cutoff=1, unk_label="<unk>") where { T <: AbstractString}
+
+Initiate Type for providing MLE ngram model scores.
+
+Implementation of Base Ngram Model.
+   
+"""
+function MLE(word, unk_cutoff=1, unk_label="<unk>")
```

Comment:
This is highly unlikely, but we can do a sanity check for "<unk>" being in vocab itself and raising error when this happens. 
Also add to tests.

Reply:
It is already there ayush in Vocabulary.jl

Reply:
Oh. Sorry I missed that one. Thanks. Marking this resolved.